js/main.js

```diff
@@ -375,7 +375,7 @@ jQuery(document).on('ready', function() {
       var talk_id = schedule[i].talk_id;
       var entity_details = schedule[i];
       var title = entity_details.title;
-      var description = tracks[talk_id].description;
+      var description = markdownit({breaks:true}).render(tracks[talk_id].description.replace(/\\n/g,"\n"));
       var speaker_name = tracks[talk_id].hasOwnProperty('speaker') ? tracks[talk_id].speaker.name : '';
       var time_duration = entity_details.start_time + ' - ' + entity_details.end_time;
       var display_title = speaker_name !== '' && typeof speaker_name !== 'undefined' ? title + ' by ' + speaker_name : title;
```

Comment:
Delete this line.

Reply:
This line converts markdown text to HTML. Are you sure I shall remove this?